### PR TITLE
Display offers

### PR DESCRIPTION
**Pull request by mitia-anah**
Hey Team,

The display offers is ready done, and it is ready to be reviewed. Have a look at it and leave some comments if it is necessary.

- Clickup: https://app.clickup.com/t/41k871
- Ref#41k871

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
-.App {
+/* .App {
   text-align: center;
-}
+} */
```
Comment (lightme-fan) [inline src/App.css:3]: Delete this if not used.

File: src/components/Offer/Offer.stories.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ export default {
 const Template: Story<OfferProps> = (args) => <Offer {...args} />
 export const OfferComponent = Template.bind({})
 OfferComponent.args = {
-  imageForOffer: Image,
+  // imageForOffer: Image,
```
Comment (lightme-fan) [inline src/components/Offer/Offer.stories.tsx:16]: Delete this comment.

File: src/components/Offer/Offer.tsx [hunk 3]
```diff
@@ -38,6 +38,7 @@ const OfferStyle = styled.div`
         display: none;
       }
     }
+
     .favoriteButtonContainer {
       display: none;
       @media (min-width: 1114px) {
```
Comment (lightme-fan) [inline src/components/Offer/Offer.tsx:44]: Use the new mediaQueries methods that Andrej suggested.

[https://codesandbox.io/s/media-queries-in-styled-components-typescript-forked-o4tt0](https://codesandbox.io/s/media-queries-in-styled-components-typescript-forked-o4tt0)

**Review by lightme-fan — COMMENTED** [review]
Looks good!

We still have to fix the media queries later.

**Review by lightme-fan — APPROVED** [review]

**Review by Tiaraha-funny — APPROVED** [review]

File: src/components/OffersList/OffersList.tsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+      {offers.map((offer) => (
+        <Offer
+          key={offer.id}
+          offerName={offer.title}
+          amount={25}
+          amountOfProduct={27}
+          location={{ city: 'Mahanoro', country: 'Madagascar' }}
+          currency={''}
+          detailButtonText='See details'
+          imageDescription={''}
```
Comment (andrzej-onja) [inline src/components/OffersList/OffersList.tsx:44]: image description is not needed

File: src/redux/slices/adsSlice.tsx [hunk 5]
```diff
@@ -14,14 +14,14 @@ type FetchAdsError = {
...
-  number,
+  string[],
   { rejectValue: FetchAdsError }
 >(
   'ads/fetch',
   // The second argument, `thunkApi`, is an object
   // that contains all those fields
   // and the `rejectWithValue` function:
-  async (limit: number, thunkApi) => {
+  async (limit: string[], thunkApi) => {
```
Comment (andrzej-onja) [inline src/redux/slices/adsSlice.tsx:24]: dont think type for limit needs to change

File: src/components/Offer/Offer.tsx [hunk 6]
```diff
@@ -34,73 +35,85 @@ const OfferStyle = styled.div`
...
-  }
+    max-width: 800px;
+  `}
 `
-
+interface Photo {
+  url: string
+  info: string
+  isPrimary: boolean
+}
```
Comment (andrzej-onja) [inline src/components/Offer/Offer.tsx:114]: reuse Photo interface defined in OfferList (there is a comment there about adding an export)

**Review by andrzej-onja — APPROVED** [review]

**Pull request merged**